tests/tas-elasticsearch/pom.xml

```diff
@@ -51,9 +51,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.alfresco</groupId>
-            <artifactId>alfresco-elasticsearch-shared</artifactId>
-            <version>${alfresco-elasticsearch-shared.version}</version>
+            <groupId>org.elasticsearch.client</groupId>
```

Comment:
Please remove this dependency and the tests that rely on it.

Reply:
done